pkgs/development/compilers/gcc/8/default.nix

```diff
@@ -316,6 +317,10 @@ stdenv.mkDerivation ({
       "--with-gnu-as" "--without-gnu-ld"
     ]
     ++ optional (targetPlatform == hostPlatform && targetPlatform.libc == "musl") "--disable-libsanitizer"
+    ++ optionals (targetPlatform.isPower && targetPlatform.libc == "glibc") [
+      "--with-long-double-128"
+      "--enable-softfloat"
+    ]
```

Comment:
Don't forget to update pkgs/development/compilers/gcc/snapshot/default.nix too

Reply:
Yeah it's some arbitrary vcs revision. Should be newer than the rest so it would become the next release version, GCC 9 here.